tests/gtest/avifincrtest_helpers.h

```diff
@@ -11,6 +11,41 @@
 namespace avif {
 namespace testutil {
 
+//------------------------------------------------------------------------------
+// Duplicated from internal.h
+// Used for debugging. Define AVIF_BREAK_ON_ERROR to catch the earliest failure
+// during encoding or decoding.
+#if defined(AVIF_BREAK_ON_ERROR)
+static inline void avifBreakOnError() {
+  // Same mechanism as OpenCV's error() function, or replace by a breakpoint.
+  int* p = NULL;
+  *p = 0;
+}
+#else
+#define avifBreakOnError()
+#endif
+
+// Used instead of CHECK if needing to return a specific error on failure,
+// instead of AVIF_FALSE
+#define AVIF_CHECKERR(A, ERR) \
+  do {                        \
+    if (!(A)) {               \
+      avifBreakOnError();     \
+      return ERR;             \
+    }                         \
+  } while (0)
+
+// Forward any error to the caller now or continue execution.
+#define AVIF_CHECKRES(A)              \
+  do {                                \
+    const avifResult result__ = (A);  \
+    if (result__ != AVIF_RESULT_OK) { \
+      avifBreakOnError();             \
+      return result__;                \
+    }                                 \
+  } while (0)
+//------------------------------------------------------------------------------
+
```

Comment:
Delete this.

Among the files that include this header, only avifincrtest_helpers.cc uses the `AVIF_CHECK*` macros. But avifincrtest_helpers.cc also includes aviftest_helpers.h.

Note: Some files (e.g., avifgainmaptest.cc) include both this header and aviftest_helpers.h. I am surprised that compilers allow duplicate macro definitions. I guess if the definitions are exactly the same, then they are allowed.

Reply:
Indeed, that's how C works: you are allowed to redefine the macro as long as it is the same: